docs/tutorials/tutorial_3k_tcr.md

```diff
@@ -507,38 +508,38 @@ ir.pl.spectratype(
 
 ### Repertoire overlaps
 
-```python
-df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
-```
+Overlaps in the adaptive immune receptor repertoire of samples or sample groups enables to pinpoint important clonotype groups, as well as to provide a measure of similarity between samples.  
+Running Scirpy's `repertoire_overlap` tool creates a matrix featuring the abundance of clonotypes in each sample. Additionally, it also computes a (Jaccard) distance matrix of samples as well as the linkage of hierarchical clustering.
 
 ```python
+df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
 df.head()
 ```
 
-```python
-ir.pl.repertoire_overlap(adata, "sample")
-```
+The distance matrix can be shown as a heatmap, while samples are reordered based on the linkage computed previously.
 
 ```python
 ir.pl.repertoire_overlap(adata, "sample", heatmap_cats=["patient", "source"])
 ```
 
-```python
-ir.pl.repertoire_overlap(adata, "sample", dendro_only=True, heatmap_cats=["source"])
-```
+A specific pair of samples can be compared on a scatterplot, where dot size corresponds to the number of clonotypes at a given coordinate.
 
 ```python
-ir.pl.repertoire_overlap(adata, "sample", pair_to_plot=["LN2", "LT2"])
+ir.pl.repertoire_overlap(adata, "sample", pair_to_plot=["LN2", "LT2"],fig_kws={"dpi": 120})
 ```
 
+### Discovering clonotypes preferentially occuring in a group
+
+Clonotypes associated with an experimental group (a given cell type, samle or diagnosis) might be important candidates as biomarkers or disease drivers. Scirpy offers `clonotype_imbalance` to rank clonotypes based on Fisher's exact test comparing the fractional presence of a given clonotype in two groups.
+
 ```python
 # Create a column in obs that gives us the site of the tumor
 
 adata.obs["site"] = adata.obs["patient"].str.slice(stop=-1)
 ```
 
 ```python
-# Show clonotypes that are the most imbalance between Tumor and Control samples based on Fischer's test
+# Show clonotypes that are most imbalanced between Tumor and Control samples based on Fisher's test
 
 ir.pl.clonotype_imbalance(
```

Comment:
This is probably not for now due to the time constraints... but Is there any way of visualizing this in a more compact way?  Maybe a heatmap? Somehow a lot of space is wasted with a two-axis panel while most of the dots are zero anyway.

Reply:
Feel free to turn that into an issue for later...

Reply:
Yes, let us leave it as an issue. Heatmap might be a good idea, but might not be necessary before submission